es-design-system/pages/molecules/es-popover.vue

```diff
@@ -39,7 +45,8 @@
                         </b-link>
                     </label>
                     <EsPopover
-                        target="darkTitleTarget"
+                        :placement="placement.directive"
+                        :target="`darkTitleTarget-${placement.directive}`"
                         variant="dark">
                         <template #title>
                             My Title
```

Comment:
Nit: I realize this was pre-existing on this page but all titles, all links, everywhere, should always be sentence case rather than title case - I'd like us to work towards making the design system consistent with its own guidance on this, otherwise people may copy and paste this example and do the wrong thing - can we update this to "My title", the links to "Learn more", etc. throughout this page?